object_database/web/cells/cells.py

```diff
@@ -1863,6 +1947,11 @@ def recalculate(self):
 
 
 class SubscribedSequence(Cell):
+    # TODO: Get a better idea of what is actually happening
+    # in this cell. For example, what is with all the existing
+    # items funging, and what is actually needed in terms of
+    # information to display this correctly in the new
+    # JS-component based setup?
     def __init__(self, itemsFun, rendererFun, asColumns=False):
```

Comment:
This thing calls 'itemsFun' to get a list of items and then 'rendererFun' to produce the actual cell contents for each 'item'. The 'item' is really just a way of tracking the unique things in the sequence. Three things can happen: new items, items deleted, and items reordered.